ert_shared/libres_facade.py

```diff
@@ -1,5 +1,8 @@
 import os
+import time
+import logging
 from pandas import DataFrame
+import numpy as np
```

Comment:
Is this import needed? Also, `time` seems to be imported whiteout usage.

Reply:
Thx...  left over from testing